.github/workflows/ci-python.yml

```
@@ -156,9 +156,9 @@ jobs:
           python make.py python-ci-install
 
           # Make sure POETRY_LIBPARSEC_BUNDLE_EXTRA_SHARED_LIBRARIES=false worked
-          if [ -n "$( ls -A ./server/parsec.libs 2>/dev/null )" ];
+          if [ -n "$(ls -A ./server/parsec.libs)" ];
```

What `ls` errors can occur here? I guess probably a non-existing directory which can be check explicitly:
```suggestion
          if [ -d "./server/parsec.libs" ] && [ -n "$(ls -A ./server/parsec.libs)" ];
```